k8-operator/packages/model/model.go

```diff
@@ -17,8 +17,9 @@ type RequestUpdateUpdateDetails struct {
 }
 
 type SingleEnvironmentVariable struct {
-	Key   string `json:"key"`
-	Value string `json:"value"`
-	Type  string `json:"type"`
-	ID    string `json:"_id"`
+	Key        string `json:"key"`
+	Value      string `json:"value"`
+	SecretPath string `json:"secretPath"`
+	Type       string `json:"type"`
+	ID         string `json:"_id"`
```

Comment:
Nit: While you're at it, can we change `_id` to use the `id` field?

Reply:
Updated